src/Keboola/Console/Command/DeleteOwnerlessWorkspaces.php

```diff
@@ -73,47 +73,41 @@ protected function execute(InputInterface $input, OutputInterface $output): void
                 $tokensClient->getToken($sandbox->getTokenId());
                 continue; // token exists so no need to do anything
             } catch (ClientException $exception) {
-                if ($exception->getCode() === 404) {
-                    // sandbox is owned by inactive token
-                    if ($this->shouldDeleteSandbox($sandbox, $includeShared)) {
-                        if (!in_array($sandbox->getType(), Sandbox::CONTAINER_TYPES)) {
-                            // it is a database workspace
-                            $output->writeln('Deleting inactive storage workspace ' . $sandbox->getPhysicalId());
-                            $totalDeletedStorageWorkspaces++;
-                            if ($force) {
-                                $workspacesClient->deleteWorkspace($sandbox->getPhysicalId());
-                            }
-                        } else {
-                            if (!empty($sandbox->getStagingWorkspaceId())) {
-                                $output->writeln(
-                                    'Deleting inactive staging storage workspace ' . $sandbox->getPhysicalId()
-                                );
-                                $totalDeletedStorageWorkspaces++;
-                                if ($force) {
-                                    $workspacesClient->deleteWorkspace($sandbox->getStagingWorkspaceId(), [], true);
-                                }
-                            }
-                        }
-                        $totalDeletedSandboxes++;
-                        if ($force) {
-                            $sandboxesClient->delete($sandbox->getId());
-                        }
-                    }
+                if ($exception->getCode() !== 404) {
+                    throw $e;
                 }
             }
+
+            // sandbox is owned by inactive token
+            if (!$includeShared && !$sandbox->getShared()) {
```

Comment:
I think this should be that we continue if sandbox is shared and the option to include it isn't set
```suggestion
            if (!$includeShared && $sandbox->getShared()) {
```